preview/main.go

```diff
@@ -38,20 +38,10 @@ func main() {
 	}
 
 	testdata := []data{
-		{"de4871ce-0bfd-4b4b-aee2-e02427f4e38b", string(types.WorkitemCreate)},
-		{"43024450-fe8c-4082-8828-88512cebfdb0", string(types.WorkitemCreate)},
-		{"3a331aa3-6423-4fd7-85e4-95d7932b168c", string(types.WorkitemCreate)},
-		{"d85e19a1-f4aa-486e-a8fe-3211cac9b68f", string(types.WorkitemCreate)},
-		{"43024450-fe8c-4082-8828-88512cebfdb0", string(types.WorkitemUpdate)},
-		{"d28f8344-4956-497a-b43b-7f217087a931", string(types.CommentCreate)},
-		{"51d968b1-b9e5-4ec1-884a-ff256902c753", string(types.CommentCreate)},
-		{"51d968b1-b9e5-4ec1-884a-ff256902c753", string(types.CommentUpdate)},
-		{"3383826c-51e4-401b-9ccd-b898f7e2397d", string(types.UserEmailUpdate)},
-		{"f3a184bf-963b-448a-86b4-307ae455223b", string(types.UserDeactivation)},
-		{"81d1c3bf-fcf2-4c4e-9d12-f9e5c15fb9ab", string(types.InvitationTeamNoorg)},
-		{"297f2037-72e9-42b3-a5fc-76d843877163", string(types.InvitationSpaceNoorg)},
-		{"0a9c6814-462e-411c-8560-d74297bf1ceb", string(types.AnalyticsNotifyCVE)},
-		{"7dd737ed-c0fa-4f2f-9a3a-ed43778bb7fe", string(types.AnalyticsNotifyVersion)},
+		{"1a3496ca-edc0-42f7-958f-ba02ed3ef54d", string(types.UserEmailUpdate)},
```

Comment:
why do you use the same value everywhere?

Reply:
I’m using the same identity since it’s easier. Is it a problem?

Reply:
not really. I was just curious.